app/views/StatusView/index.tsx

```diff
@@ -71,18 +73,23 @@ const Status = ({
 }) => {
 	const { id, name } = statusType;
 	return (
-		<List.Item
-			title={name}
-			onPress={() => {
-				const key = `STATUS_${statusType.id.toUpperCase()}` as keyof typeof events;
-				logEvent(events[key]);
-				if (status !== statusType.id) {
-					setStatus(statusType.id);
-				}
-			}}
-			testID={`status-view-${id}`}
-			left={() => <StatusIcon size={24} status={statusType.id} />}
-		/>
+		<>
+			<List.Item
+				additionalAcessibilityLabel={` - ${status === id ? I18n.t('Current_Status') : ''}`}
```

Comment:
Is the dash needed?

Reply:
i dont think so, it is only to let it a little bit slower but we talked about it yesterday.
Removed :)